### DISCUSSION TITLE
What is the replacement for RequestedTheme

### DISCUSSION THREAD

**Discussion by mysteryx93** [AvaloniaUI/Avalonia#10438] [Q&A]
Before I'd do

    AvaloniaLocator.Current.GetService<FluentAvaloniaTheme>().RequestedTheme = "Dark"

That still works but `RequestedTheme` is marked as obsolete.

What is now the replacement for it, to allow the user to specify the theme?

**Comment by maxkatz6** [comment]
Is it FluentAvalonia type?
Anyway, it's Application.Current.RequestedThemeVariant. Or per window.
